### DISCUSSION TITLE
Altering the config

### DISCUSSION THREAD

**Discussion by TeeDDeeT** [ORelio/Minecraft-Console-Client#2057] [General]
How can I alter the config so the Client can work on a server supporting other Minecraft versions?

**Comment by ORelio** [comment]
Set version in configuration file:
https://github.com/MCCTeam/Minecraft-Console-Client/blob/17dbe10ff3c3fc344e7f6bfa82607340eebeb005/MinecraftClient/Resources/config/MinecraftClient.ini#L26
